src/services/emai.services.js

```diff
@@ -0,0 +1,88 @@
+import nodemailer from 'nodemailer';
+import 'dotenv/config';
+import jwt from 'jsonwebtoken';
+
+const transporter = nodemailer.createTransport({
+  host: process.env.SMTP_HOST,
+  port: process.env.SMTP_PORT,
+  auth: {
+    user: process.env.SMTP_USER,
+    pass: process.env.SMTP_PASSWORD,
+  },
+});
+
+function send({ email, subject, html }) {
+  return transporter.sendMail({
+    to: email,
+    subject,
+    html,
+  });
+}
+
+function sendActivationEmail(email, token) {
+  const href = `http://localhost:3005/activation/${token}`;
+  const html = `
+  <h1>Activate account</h1>
+  <a href="${href}">${href}</a>
+  `;
+
+  return send({
+    email,
+    html,
+    subject: 'Activate',
+  });
+}
+
+function sendResetPasswordEmail(email, token) {
+  const resetUrl = `http://localhost:3005/resetPassword/${token}`;
```

Comment:
The URL 'http://localhost:3005/resetPassword/' is hardcoded. Consider using an environment variable for the base URL to make it configurable for different environments.